docs/wiki/99_Developers/Update_LORIS.md

```diff
@@ -31,4 +31,18 @@ Consult the [Release notes](https://github.com/aces/Loris/releases) for the Upgr
     ALTER TABLE users CONVERT TO CHARACTER SET utf8;
  ```
 ## Imaging projects
-Update your [Loris-MRI code](https://github.com/aces/Loris-MRI/releases) to the latest Loris-MRI release and carefully review the Loris-MRI release notes for changes to the imaging pipeline scripts, config files and tables, and follow all recommended steps for Existing Projects. We recommend testing your pipelines after upgrading.
\ No newline at end of file
+Update your [Loris-MRI code](https://github.com/aces/Loris-MRI/releases) to the latest Loris-MRI release and carefully review the Loris-MRI release notes for changes to the imaging pipeline scripts, config files and tables, and follow all recommended steps for Existing Projects. We recommend testing your pipelines after upgrading.
+
+### Troubleshooting 
+In the latest release, Loris needs to upgrade the PHP version. If you have an old PHP version installed, you need to upgrade to the higher PHP version. (example: PHP 7.3 to PHP 7.4 in Ubuntu)
+In order to ensure that this new PHP version is installed correctly, you need to do the following steps: 
```

Comment:
This is more relevant on the 23 branch since 23 requires php 7.4 - but let's merge because it will apply for anyone upgrading from 1+ release ago.